test-executor/src/main/java/com/cognifide/aet/executor/common/SuiteFactory.java

```diff
@@ -26,21 +26,21 @@
 import com.cognifide.aet.communication.api.metadata.Suite;
 import com.cognifide.aet.communication.api.metadata.Test;
 import com.cognifide.aet.communication.api.metadata.Url;
-import com.cognifide.aet.model.CollectorStep;
-import com.cognifide.aet.model.ComparatorStep;
-import com.cognifide.aet.model.ExtendedUrl;
-import com.cognifide.aet.model.ParametrizedStep;
-import com.cognifide.aet.model.TestRun;
-import com.cognifide.aet.model.TestSuiteRun;
+import com.cognifide.aet.executor.model.CollectorStep;
+import com.cognifide.aet.executor.model.ComparatorStep;
+import com.cognifide.aet.executor.model.ExtendedUrl;
+import com.cognifide.aet.executor.model.ParametrizedStep;
+import com.cognifide.aet.executor.model.TestRun;
+import com.cognifide.aet.executor.model.TestSuiteRun;
 
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.List;
 import java.util.Map;
 
-class SuiteFactory {
+public class SuiteFactory {
```

Comment:
SuiteFactory is used only by SuiteExecutor. Maybe it should be left in the same package with the package-scope?
(I may be wrong)
(should have)